terraform/cross-device/workload_identity.tf

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module "project-iam-bindings" {
+  source   = "terraform-google-modules/iam/google//modules/projects_iam"
+  version  = "7.6.0"
+  projects = [data.google_project.project.project_id]
+
+  bindings = {
+    "roles/spanner.admin"                  = var.list_nodepool_sa_iam_emails,
+    "roles/logging.logWriter"              = var.list_nodepool_sa_iam_emails,
+    "roles/iam.serviceAccountTokenCreator" = var.list_nodepool_sa_iam_emails,
+    "roles/storage.objectAdmin"            = var.list_nodepool_sa_iam_emails,
+    "roles/pubsub.admin"                   = var.list_nodepool_sa_iam_emails
+  }
+}
```

Comment:
This is not going to work because we [set authoritative bindings for these service accounts](https://github.com/GoogleCloudPlatform/federated-learning/blob/main/terraform/iam.tf#L29).

Also, are you sure that you need to grant those roles to the nodes service accounts, and not the service accounts you're going to map Kubernetes service accounts to? See an example of this (`fl-workload-identity` and `cloud-source-repositories-workload-identity` in https://github.com/GoogleCloudPlatform/federated-learning/blob/main/terraform/iam.tf)